samples/boards/nrf/nrf2_clock_control/Kconfig

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+config SAMPLE_CLOCK_FREQUENCY_HZ
+	int "Frequency specification to request from clock in Hz"
+
+config SAMPLE_CLOCK_ACCURACY_PPM
+	int "Accuracy specification to request from clock in PPM"
+
+config SAMPLE_CLOCK_PRECISION
+	int "Precision specification to request from clock"
```

Comment:
Wouldn't it make sense to use `default 0` for these options (the API defines that 0 means: no specific requirement) and change them only when a certain value is required?

Reply:
added default 0 along with help to indicate what values of 0, 1 and >1 means